### PR TITLE
chore: update saas git url

### PR DESCRIPTION
**Pull request by hamidzr**
## Description
[DET-7934]

## Commentary (optional)

@keita-determined  couldn't clone with the previous url without a PAT

## Checklist

- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.
- [ ] If modifying `/webui/react/src/shared/` verify `make -C webui/react test-shared` passes.

[DET-1234]: https://determinedai.atlassian.net/browse/DET-1234?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[DET-7934]: https://determinedai.atlassian.net/browse/DET-7934?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: webui/react/src/shared/bin/test.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 parser = argparse.ArgumentParser()
 
 SHARED_WEB_REPO='https://github.com/determined-ai/shared-web'
-SAAS_REPO='ssh://git@github.com/determined-ai/saas'
+SAAS_REPO='git@github.com:determined-ai/saas'
```
Comment (keita-determined) [inline webui/react/src/shared/bin/test.py:10]: Probably we should add `.git` after `saas` like `saas.git`
https://stackoverflow.com/a/8686727

**Review by keita-determined — APPROVED** [review]

**Comment by keita-determined** [comment]
Thank you @hamidzr

File: webui/react/src/shared/bin/test.py [hunk 2]
```diff
@@ -6,9 +6,9 @@
 
 parser = argparse.ArgumentParser()
 
-SHARED_WEB_REPO='https://github.com/determined-ai/shared-web'
-SAAS_REPO='git@github.com:determined-ai/saas'
-CORE_REPO='https://github.com/determined-ai/determined'
+SHARED_WEB_REPO='https://github.com/determined-ai/shared-web.git'
+SAAS_REPO='git@github.com:determined-ai/saas.git'
+CORE_REPO='https://github.com/determined-ai/determined.git'
```
Comment (keita-determined) [inline webui/react/src/shared/bin/test.py:11]: nice

**Comment by hamidzr** [comment]
the failed test is known and unrelated https://determined-ai.slack.com/archives/C6D7MDSQ5/p1660249788633709

**Pull request merged**